packages/neon/neon_notifications/lib/pages/main.dart

```diff
@@ -26,24 +26,33 @@ class _NotificationsMainPageState extends State<NotificationsMainPage> {
   @override
   Widget build(final BuildContext context) => ResultBuilder<List<NotificationsNotification>>.behaviorSubject(
         stream: bloc.notifications,
-        builder: (final context, final notifications) => Scaffold(
-          resizeToAvoidBottomInset: false,
-          floatingActionButton: FloatingActionButton(
-            onPressed: () async {
-              bloc.deleteAllNotifications();
-            },
-            tooltip: AppLocalizations.of(context).notificationsDismissAll,
-            child: const Icon(MdiIcons.checkAll),
-          ),
-          body: NeonListView(
-            scrollKey: 'notifications-notifications',
-            isLoading: notifications.isLoading,
-            error: notifications.error,
-            onRefresh: bloc.refresh,
-            itemCount: notifications.data?.length,
-            itemBuilder: (final context, final index) => _buildNotification(context, notifications.data![index]),
-          ),
-        ),
+        builder: (final context, final notifications) {
+          final unreadCounter = bloc.unreadCounter.value;
```

Comment:
This isn't necessary since you only use it for `initialData` and it will have the same value as the stream. Since it is a BehaviourSubject the current value will be emitted when the listener attaches.